styles.css

```diff
@@ -1,3 +1,793 @@
-* {
-    font-size: 1.5rem;
+body {
+    color: #fff;
+    background: #0D0D11;
+    text-align: center;
+    margin: auto;
+    width: 1920px;
+    height: 3288px;
+}
+
+li a {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 16px;
+    font-weight: 400;
+    line-height: 24px;
+    list-style: none;
+    text-decoration: none;
+    color: #fff;
+}
+
+.toggle-button {
+    padding: 0;
+    border: 0;
+    height: 48px;
+    background-color: transparent;
+    cursor: pointer;
+}
+
+.discord-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 8px;
+    font-size: 16px;
+    color: #fff;
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    line-height: 24px;
+    padding: 12px 24px;
+    border: none;
+    width: 147px;
+    height: 48px;
+    background-color: #141B22;
+    cursor: pointer;
+}
+
+.connect-button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 8px;
+    font-size: 16px;
+    color: #000;
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    line-height: 24px;
+    padding: 12px 24px;
+    border: none;
+    height: 48px;
+    width: 149px;
+    background: #21E786;
+    cursor: pointer;
+}
+
+.breadcrumbs-nav a {
+    font-size: 14px;
+    color: #888B8E;
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    line-height: 20px;
+}
+
+.main-heading,
+.heading {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-weight: 400;
+    font-size: 44px;
+    line-height: 56px;
+    color: #fff;
+    margin: 0;
+}
+
+.sub-heading,
+.how-to-text {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 24px;
+    font-weight: 400;
+    line-height: 32px;
+    color: #fff;
+}
+
+.how-to-heading {
+    font-family: "Bakbak One", Arial, sans-serif;
+    font-size: 22px;
+    font-weight: 400;
+    line-height: 30px;
+    text-align: center;
+    color: #21E786;
+
+}
+
+.ot-member-pos,
+.faq-description,
+.joc-text,
+.sub-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-weight: 400;
+    font-size: 18px;
+    line-height: 28px;
+    color: #C2C3C5;
+}
+
+.copyright-text {
+    font-family: Poppins, Arial, sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+    line-height: 22px;
+
+}
+
+.visually-hidden {
+    visibility: hidden;
+}
+
+.nft-card,
+.about-us-list-content,
+.step-list,
+.roadmap-phase-list,
+.roadmap-nav-list,
+.faqs-list,
+.social-links-logo,
+.social-links,
+.filter-list,
+.catalog-list,
+.blog-list,
+.tag-list,
+.social-list,
+.categories-list,
+.recent-post-list {
+    list-style: none;
```

Comment:
Instead of listing out all the different lists for styling, you can target the ul tag directly.

When you want to remove list styling (e.g., list-style: none), it's more efficient to target the ul or ol tags directly, rather than listing each individual class. This approach simplifies your CSS and makes it easier to maintain, especially as your project grows.